README.md

```diff
@@ -1,7 +1,161 @@
+distributed by Jolla. Depending on the type of software, recommended
+ways of submission vary, as described below.
+
+After the successful submission, updates are handled by changing
+version information at OBS service file in `sailfishos:chum:testing`,
+checking that it compiles, and promotion to `sailfishos:chum`.
+
+
+### Asking for help
+
+If something is not clear or you get stuck in the process, feel free
+to ask for help via IRC (#sailfishos),
+[Sailfish forum](https://forum.sailfishos.org), or filing
+issues over here.
+
+You can ask someone else to package the software as well. Just be
+aware that for this effort to work out, we need many to contribute.
+
+
+### Submitting actively developed applications
+
+If you have the application compiled at OBS already, just use "Submit
+package" for the application and all the libraries it requires.
+
+If you have not developed using OBS yet, then you could either ask
+maintainers to add your project to OBS directly into
+`sailfishos:chum:testing` or, a better long-term solution, make
+account at OBS through pinging `lbt` at IRC and configure the package
+at OBS.
+
+For actively developed applications, it is expected that the software
+is compiled from the main application repository where it is
+developed. No forks at GitHub into https://github.com/sailfishos-chum
+are expected.
+
+
+### Submitting abandoned applications
+
+Assuming that the application has not been picked up by some
+developer, it is suggested to open an issue in this repository with
+the request to add that application into the repository. Include
+original GitHub/GitLab repository link with the issue.
+
+The plan is then to make a fork of the application into this GitHub
+organization (https://github.com/sailfishos-chum), if needed, update
+packaging scripts and then add it to `sailfishos:chum:testing` with
+the later promotion to `sailfishos:chum`.
+
+
+### Submitting Linux tools and libraries
+
+If you want to submit tool or library which is mainly upstream version
+(for example Midnight Commander) then it is suggested to proceed by
+making repository in this organization and later adding it to OBS. For
+that, open an issue in this repository, maintainers will create
+repository for you and then you would be able to package the software
+as needed.
+
+If you have packaging repository already, it is possible to either
+fork it or, if you wish, you could transfer the repository over to
+this organization.
+
+As for the packaging approach, it is suggested to use the same
+approach as is used for the most of Sailfish OS core packages: adding
+upstream as submodule and packaging scripts/patches into the packaging
+repository.
+
+
+## Q&A
+
+- My software requires specific SFOS versions, can I still use Chum
+  repositories? Yes, you can. You would just have to disable the
+  versions and architectures of SFOS that are not supported in OBS
+  package Meta settings. Example: see Pure Maps.
```

Comment:
Should we mention the macros?

Reply:
Yes, we should. Although they are so far defined for 4.1.0.24 only, if I remember correctly. something we will have to ask lbt to fix.

Reply:
No, theyre here too https://build.sailfishos.org/project/prjconf/sailfishos:4.0.1.48

Reply:
Not in https://build.sailfishos.org/project/prjconf/sailfishos:3.4.0.24 3.3.0.16 3.2.1.20

Reply:
For clarity: macros are a separate question. Namely, if you need to adjust something in SPEC then they will help. Here, case is when you cannot get it working for earlier versions at all. For Pure Maps, question is in gcc version required by Mapbox GL QML plugin.

Do you want to write Q&A for macros or should I fill it? I can probably do it in ~2 hours